app/repository/modifier.py

```diff
@@ -24,7 +24,7 @@ def add_label(label, message, user, session=db.session):
             return True
         except Exception as e:
             session.rollback()
-            logger.error(e)
+            logger.error('Error retrieving messages from database', error=e)
```

Comment:
More context about what it's doing (as the same message seems to be fired from multiple places?).